### PR TITLE
policies: initial pass over IT related policies

### PR DESCRIPTION
**Pull request by adamdecaf**
Ideas
- https://about.gitlab.com/handbook/engineering/

### CODE REVIEW

**Comment by adamdecaf** [comment]
Here's another public example: https://github.com/ridem/comply-starter-netlify-cms

File: narratives/security.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Describe product infrastructure, emphasizing security measures
 
 {{.Name}} uses the following cloud services for its internal infrastructure:
 
-- List cloud services
+- Google Cloud Platform TODO(adam): link to their policy page?
```
Comment (wadearnold) [inline narratives/security.md]: Do we need to define GitHub and Google G Suite as providers as well?

File: narratives/security.md [hunk 2]
```diff
@@ -46,14 +46,14 @@ Access to these cloud services is limited according to the role of the {{.Name}}
 
 - operating system must be within one generation of current
 - full-disk encryption
-- onboard antivirus/antimalware software
+- antivirus/antimalware software
```
Comment (wadearnold) [inline narratives/security.md]: Do we need to run an antivirus on our cloud servers?

File: narratives/security.md [hunk 3]
```diff
@@ -46,14 +46,14 @@ Access to these cloud services is limited according to the role of the {{.Name}}
...
 - OS and AV automatically updated
 
 Workstation compliance with these measures is evaluated on a quarterly basis.
 
 ## Remote Access
 
-Many {{.Name}} employees work remotely on a regular basis and connect to production and internal IT systems via the same methods as those employees connecting from the {{.Name}} physical office, i.e., direct encrypted access to cloud services. It is the employee's responsibility to ensure that only authorized personnel use {{.Name}} resources and access {{.Name}} systems.
+All {{.Name}} employees work remotely on a regular basis and connect to production and internal IT systems via secure protocols. It is the employee's responsibility to ensure that only authorized personnel use {{.Name}} resources and access {{.Name}} systems in their physical locations.
```
Comment (wadearnold) [inline narratives/security.md]: I wonder if we need to add ubikey or other access tokens to ensure remote workers are who they say they are? This is good for now but we should probably double-check against gitlab and others.

Comment (wadearnold) [reply narratives/security.md]: Maybe we should just have a policy that requires multifactor authentication. Ensure that is turned on for github, g-suite, etc

File: policies/access.md [hunk 4]
```diff
@@ -6,52 +6,49 @@ satisfies:
...
-    i. HR and IT will follow the same steps as outlined in the onboarding and offboarding procedures. 
 
-a. *Review of accounts and permissions:*
-
-    i. Each month, IT and HR will review accounts and permission levels for accuracy. 
+    i. HR and IT will follow the same steps as outlined in the onboarding and offboarding procedures.
 
+a. *Review of accounts and permissions:*
 
+    i. Each month, IT and HR will review accounts and permission levels for accuracy.
```
Comment (wadearnold) [inline policies/access.md]: We will need to keep track of these reports in a google docs folder for each month.

File: policies/disaster.md [hunk 5]
```diff
@@ -24,90 +24,94 @@ a. The following conditions must be met for this plan to be viable:
...
+    i. CEO is authorized to cooperate with all contracted vendors.
 
 a. Specific recovery steps for information systems infrastructure and services are provided in Appendix B.
 
-\pagebreak 
+\pagebreak
 
 # Appendix A: Relevant Points of Contact
 
+TODO(adam): can/should we redact this somehow?
```
Comment (wadearnold) [inline policies/disaster.md]: I think this section will expand quickly this year.

File: narratives/security.md (hunk 2 above)
Comment (adamdecaf) [reply narratives/security.md]: Probably? We use the default image for GKE (Google's Hosted Kubernetes). If we need to run AV we'll create a base image with it installed.

File: narratives/security.md (hunk 1 above)
Comment (adamdecaf) [reply narratives/security.md]: Yep. Added

File: narratives/security.md (hunk 3 above)
Comment (adamdecaf) [reply narratives/security.md]: I'd like to have a policy requiring hardware tokens for GitHub and G Suite.

**Pull request closed without merging**